### ISSUE TITLE
Webpack 5 invalid configuration 

### ISSUE THREAD

**Issue by hanford** [martpie/next-transpile-modules#162]
- [x] I HAVE READ [THE FAQ](https://github.com/martpie/next-transpile-modules#faq) AND MY PROBLEM WAS NOT DESCRIBED THERE
- [x] I WILL GIVE 10$ TO CHARITY IF MY SOLUTION WAS ACTUALLY IN THE README/FAQ

**Describe the bug**
With the latest version of Next, v10.0.6 .. I am now seeing an invalid Webpack 5 configuration which comes from this package.

```
WebpackOptionsValidationError: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.

 - configuration[0] has an unknown property 'snapshot'. These properties are valid:
   object { amd?, bail?, cache?, context?, dependencies?, devServer?, devtool?, entry?, externals?, infrastructureLogging?, loader?, mode?, module?, name?, node?, optimization?, output?, parallelism?, performance?, plugins?, profile?, recordsInputPath?, recordsOutputPath?, recordsPath?, resolve?, resolveLoader?, serve?, stats?, target?, watch?, watchOptions? }
```

**To Reproduce**
Steps to reproduce the behavior (a reproduction is even better):

Reproduction: https://github.com/hanford/ntm-bug

**Setup**

- Next.js version: 10.0.6
- `next-transpile-modules` version: 6.1.0
- Node.js version: n/a
- `npm`/`yarn` version: n/a
- Operating System: Mac OS Big Sur
- Webpack 4 or 5: 5

**Comment by martpie** [comment]
Please downgrade to 10.0.5, we have a WIP to fix this bug, should be released in the next days

**Comment by hanford** [comment]
We downgraded for other reasons, thanks @martpie feel free to close this out either now (if it's a duplicate) or when the next version is released

**Comment by martpie** [comment]
Should be fixed in 6.2.0, cheers!
